Base/QTGUI/qSlicerSaveDataDialog.cxx

```diff
@@ -1296,6 +1402,17 @@ void qSlicerSaveDataDialogPrivate::updateOptionsWidget(int row)
   this->FileWidget->resizeColumnToContents(OptionsColumn);
 }
 
+//-----------------------------------------------------------------------------
+void qSlicerSaveDataDialogPrivate
+::updateUserMessagesItem(int row)
+{
+vtkMRMLStorableNode * const node = vtkMRMLStorableNode::SafeDownCast(this->object(row));
+  QString allUserMessages = makeUserMessages(node);
+  QTableWidgetItem* const userMessagesItem = this->FileWidget->item(row, UserMessagesColumn);
+  Q_ASSERT(userMessagesItem);
```

Comment:
Remove assert, if we expect such error ... a proper warning should be reported to the user.

Reply:
I don't expect that a user could cause such an error; but a pull request could.  I will remove it.